lib/salemove/process_handler/pivot_process.rb

```diff
@@ -208,11 +208,7 @@ def handle_exception(exception, input)
           @logger.error(message, input)
 
           if @exception_notifier
-            @exception_notifier.notify_or_ignore(
-              exception,
-              cgi_data: ENV.to_hash,
-              parameters: input
-            )
+            @exception_notifier.notify_or_ignore(exception, ENV.to_hash.merge(input))
```

Comment:
Is it correct to mix ENV with params?

Reply:
Airbrake API defines separate fields for params and environment: https://airbrake.io/docs/api/#create-notice-v3

Reply:
`environment` is populated by the client like this: https://github.com/airbrake/airbrake-ruby/blob/510f1990e312b970a4f5375f14e4b11cdb1b1d73/lib/airbrake-ruby/notice.rb#L64:L66

e.g.
```
"environment": {
    "program_name": "[Filtered]"
  }
```

It seems that the new client doesn't allow providing ENV data as a separate param anymore. At least I couldn't find any proof from the source/docs.

We could either
- mix ENV with params (not sure anymore if a good idea)
- or drop including ENV in the reports (personally I have never found them to be useful anyway)
- or ???

Reply:
I'd drop ENV from params

Reply:
958d950